src/index.js

```diff
@@ -4,6 +4,7 @@ const defaultOptions = {
     baseUri: '',
     extensions: ['.gif', '.jpeg', '.jpg', '.png', '.svg'],
     flatten: false,
+    envName: 'ASSET_BASE_URI'
```

Comment:
I would call that omething else.

```suggestion
    envName: 'ASSET_BASE_URI'
    baseUriFrom: 'ASSET_BASE_URI'
```

Reply:
I should have looked at the tests. should be fine now.